prow/plugins/plugins.go

```diff
@@ -291,6 +291,25 @@ func (pa *ConfigAgent) Load(path string, supplementalPluginConfigDirs []string,
 				return nil
 			}
 
+			relPath, err := filepath.Rel(supplementalPluginConfigDir, path)
+			if err != nil {
```

Comment:
Please remove all the changes here. Both the supplemental file loading is (here) and the org repo directory structure validation (checkconfig) are already implemented.